flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-paimon/src/test/java/org/apache/flink/cdc/connectors/paimon/sink/PaimonMetadataApplierTest.java

```diff
@@ -137,6 +140,12 @@ public void testApplySchemaChange(String metastore)
         Assertions.assertEquals(
                 tableSchema, catalog.getTable(Identifier.fromString("test.table1")).rowType());
 
+        Assertions.assertEquals(
+                "col3DefValue",
+                catalog.getTable(Identifier.fromString("test.table1"))
+                        .options()
+                        .get("fields.col3.default-value"));
+
         Map<String, String> nameMapping = new HashMap<>();
         nameMapping.put("col2", "newcol2");
         nameMapping.put("col3", "newcol3");
```

Comment:
It turns out that we should modify the options of default value too if we rename the columns.

Reply:
wired design，It means that we need to update default value twice for paimon ?

Reply:
This is because Paimon saves column information through table options, and when renaming or dropping columns, we also need to modify these options accordingly.

for example, for renaming col3 to newcol3, we should modify the option like: 
fields.col3.default-value: col3DefValue => fields.newcol3.default-value: col3DefValue

Reply:
Thanks @lvyanquan for the clarification, I guessed Paimon's underlying implementation, just comment it's a little wired

Reply:
Thanks @lvyanquan for the clarification, I guessed Paimon's underlying implementation, just comment it's a little wired